rocketmq-spring-boot/src/main/java/org/apache/rocketmq/spring/support/RocketMQUtil.java

```diff
@@ -326,4 +326,10 @@ public static DefaultLitePullConsumer createDefaultLitePullConsumer(String nameS
 
         return litePullConsumer;
     }
+
+    public static String getNamespace(String specifiedNamespace, String defaultNamespace) {
+        // prefer to use annotation namespace
+        // if is empty a default namespace will be used
+        return !StringUtils.hasLength(specifiedNamespace) && StringUtils.hasLength(defaultNamespace) ? defaultNamespace : specifiedNamespace;
```

Comment:
How about using the _isBlank_ method to avoid the influence brought by whitespace?